app/src/main/res/layout/onboarding_spotlight_content_services_logo_man.xml

```diff
@@ -1,5 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<ImageView xmlns:android="http://schemas.android.com/apk/res/android"
-    android:layout_width="wrap_content"
-    android:layout_height="wrap_content"
-    android:src="@drawable/logo_man_copy_3"/>
```

Comment:
Could we remove the `logo_man_copy_3` asset?

Reply:
fixed in the last commit, thanks for mention